ramls/entity.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Entity associated with a note",
+  "additionalProperties": false,
+  "properties": {
+    "id": {
+      "type": "string",
+      "description": "Id of entity linked to a note",
+      "example": "1234-5678"
+    },
+    "type": {
+      "type": "string",
+      "description": "Type of entity",
+      "example": "package"
+    },
+    "domain": {
```

Comment:
From my understanding, domain and entity both should be part of "note". Entity will have an id and type as defined above. Thoughts?

Reply:
Sure -- makes sense that entity is confusing here -- in this case the entity is more of a link or a reference to the associated object. I can rename to link (but I still need to include domain)

Reply:
agreed. 👍